<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startScan</class>
 <widget class="QDialog" name="startScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,1,1,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Enable HDR</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0">
     <item>
      <widget class="QLabel" name="label_turnableTimes">
       <property name="text">
        <string>Turnable times</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_turnableTimes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_point">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_subScanType">
       <property name="text">
        <string>Subscan type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_subScanType">
       <item>
        <property name="text">
         <string>SST_FIXED_FREE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SST_FIXED_TURNABLE</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_alignType">
       <property name="text">
        <string>Align type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_alignType">
       <item>
        <property name="text">
         <string>AT_CODE_POINT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AT_FEATURES</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AT_TURTABLE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AT_MARKERS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AT_GLOBAL_POINT</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Turntable Speed(1-10)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_motorSpeed">
       <property name="currentIndex">
        <number>5</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Turntable Turns</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="radioButton_oneRound">
       <property name="text">
        <string>One Turn</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_halfRound">
       <property name="text">
        <string>Half Turn</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
